stratum/hal/lib/barefoot/barefoot.bzl

```diff
@@ -19,7 +19,14 @@ def _impl(repository_ctx):
         print("SDE_INSTALL is deprecated, please use SDE_INSTALL_TAR")
         bf_sde_install_path = repository_ctx.os.environ["SDE_INSTALL"]
         repository_ctx.symlink(bf_sde_install_path, local_install_path)
-    repository_ctx.symlink(Label("@//bazel:external/bfsde.BUILD"), "BUILD")
+    ver = repository_ctx.read(local_install_path + "/share/VERSION").strip("\n")
```

Comment:
If the SDE structure ever changes, we can handle this here and still expose a unified interface.